project/src/main/java/com/google/sps/data/VolunteeringOpportunity.java

```diff
@@ -0,0 +1,122 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class VolunteeringOpportunity {
+  private final long opportunityId = 10000000L;
```

Comment:
Add a TODO here on why you need this for prototype and how you gonna replace this.

Reply:
Done.